store/event_test.go

```diff
@@ -84,6 +88,15 @@ func TestScanHistory(t *testing.T) {
 	if e != nil {
 		t.Fatalf("bad index shoud reuturn nil")
 	}
+
+	// Add a dir which will replace old event
```

Comment:
it will not replace the old event. it will append to the end. the search will scan the events list in a reverse order. so it will show up first without other constraints.